app/models/push_subscription.js

```diff
@@ -0,0 +1,21 @@
+import BaseModel from "./base/base.js";
+
+export default class PushSubscription extends BaseModel {
+  constructor(params) {
+    super(params);
+  }
+  static get collection() {
+    return "push_subscriptions";
+  }
+
+  static get visibleFields() {
+    return [
+      "_id",
+      "created_at",
```

Comment:
`user_id` is also needed, cause push subscriptions will be connected to user, e.g. a user can have multuple push subscriptions, usually - one per each device